bin/steps/python

```diff
@@ -97,6 +97,22 @@ if [ ! "$SKIP_INSTALL" ]; then
   hash -r
 fi
 
+# Heroku uses the get-pip utility maintained by the Python community to vendor Pip.
+# https://github.com/pypa/get-pip
+GETPIP="https://lang-python.s3.amazonaws.com/etc/get-pip.py"
+
+# Grab SHA from get-pip
+CHECK_SHA="$(curl -s "$GETPIP" | sha256sum | cut -d ' ' -f 1)"
```

Comment:
There's no need to download this twice. You can check the SHA of the downloaded file.